### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Tomanhez**
All new classes are from BitbagCms plugin, Only what I did is change interface in fields to `Twig\ Environment `
New services have arguments replaced from `@templating` to `@twig`

### CODE REVIEW

File: config/services.yaml [hunk 1]
```diff
@@ -30,3 +30,84 @@ services:
         arguments: ['@Twig\Extra\Intl\IntlExtension']
         tags:
             - {name: twig.extension}
+
+    bitbag_sylius_cms_plugin.twig.extension.block:
```
Comment (Zales0123) [inline config/services.yaml:34]: We should put some comments about why do we need to override these services. Maybe we can also put them in a separate file?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: php
 
-dist: trusty
+dist: xenial
```
[the remark this answers is not in the document]
Comment (Tomanhez) [reply .travis.yml:3]: Ok 👍, but it's a temporary solution. I would like to replace travis with github actions. I agree with you comments but demo is broken. I prefer merge it and next I will open separate PR to fix it?

File: composer.json [hunk 3]
```diff
@@ -20,51 +20,47 @@
...
-        "symfony/flex": "^1.2",
+        "symfony/flex": "^1.7",
         "sylius/invoicing-plugin": "~0.10",
         "sylius/refund-plugin": "^1.0.0@rc",
         "sylius/customer-order-cancellation-plugin": "^1.1",
         "sylius/customer-reorder-plugin": "^1.1",
-        "bitbag/cms-plugin": "2.3.4",
+        "bitbag/cms-plugin": "3.0.0-beta1",
         "bugsnag/bugsnag-symfony": "^1.7",
         "sylius/paypal-plugin": "^1.0@rc"
```
[the remark this answers is not in the document]
Comment (lchrusciel) [reply composer.json:35]: But lets do it in the second move

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thanks, Tomasz! :1st_place_medal:

**Pull request merged**